kickstarts/partials/post/repos.ks.erb

```diff
@@ -0,0 +1,7 @@
+# post install repos
+# yum update uses these repos to update and reinstall packages.
+# Please also add to "build time repos" around line 20
```

Comment:
This comment no longer makes sense.   Can you update to refer to the corresponding partial?